src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  width: 200px;
+  padding: 0 16px;
+  margin: 0 24px 46px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &:nth-child(5n) {
+    margin-left: 0;
+  }
+
+  &__image {
+    display: block;
+    margin: 32px auto 40px;
+    height: 134px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-main-accent;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $color-secondary;
+  }
+
+  &__review {
+    display: flex;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    align-items: center;
+    justify-content: space-between;
+    margin: 16px 0 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 0 16px;
+
+    &-label {
+      color: $color-secondary;
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 400;
+    }
+
+    &-value {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+    }
+
+  }
+}
+
+.button {
+  display: flex;
+  height: 40px;
+  background-color: $color-accent-blue;
+  border-radius: 5px;
```

Comment:
Please, organize your styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)